knexfile.js

```diff
@@ -2,8 +2,19 @@
 
 const DatabaseConfig = require('./config/database.config')
 
-module.exports = {
+const defaultConfig = {
   client: 'postgres',
+  useNullAsDefault: true,
+  migrations: {
+    tableName: 'knex_migrations',
+    directory: './db/migrations'
+  },
+  seeds: {
+    directory: './db/seeds'
+  }
+}
+
+const development = {
   connection: {
```

Comment:
If we move this to a separate object:
```js
const connection = {
   host: DatabaseConfig.host,
   ...
   database: DatabaseConfig.database
}
```
We can then include it in each of the environments and only override what we need to eg. `database` for the `test` environment:
```js
const test = {
   ...defaultConfig,
   connection: {
      ...connection,
      database: DatabaseConfig.testDatabase
   }
}